regression/goto-analyzer/value-set-function-pointers-incremented-02/main.c

```diff
@@ -0,0 +1,34 @@
+#include <assert.h>
+
+typedef int (*fptr_t)(int);
+fptr_t fun_global, fun_global_show;
+
+int f(int x)
+{
+  return x + 1;
+}
+int g(int x)
+{
+  return x;
+}
+int h(int x)
+{
+  return x - 1;
+}
+
+int main(void)
+{
+  int i;
+  // This line is needed so that g is considered as a possibility for the TOP
+  // value
+  fptr_t dummy = g;
+
+  // function pointer incremented should be top
+  fptr_t fun_incremented = f;
+  if(i)
+    ++fun_incremented;
```

Comment:
Ummm... is this incrementing a function pointer?  I'm not sure that is going to work...

Reply:
It isn't. I took this from an existing test to ensure the behaviour was consistent with it.

Reply:
OK.